update/update.sh

```diff
@@ -1,9 +1,17 @@
-#!/usr/bin/env bash
-set -euo pipefail
+#!/bin/sh
+set -eu
 
 cd /bundle_update
 
-echo "Running pre install script or command..."
-eval "$PRE_BUNDLE_UPDATE"
+if [ -n "$PRE_BUNDLE_UPDATE" ]; then
+  echo "--- :shell: Running pre bundle update"
+  eval "$PRE_BUNDLE_UPDATE"
+fi
 
+echo "+++ :bundler: Running bundle update"
 bundle update --jobs="$(nproc)"
+
+if [ -n "$POST_BUNDLE_UPDATE" ]; then
+  echo "--- :shell: Running post bundle update"
+  eval "$POST_BUNDLE_UPDATE"
```

Comment:
Here we run the post-bundle-update command, if provided in the plugin definition in the project's `pipeline.yml`.